web/components/answers/answer-resolve-panel.tsx

```diff
@@ -419,12 +422,49 @@ export const IndependentAnswersResolvePanel = (props: {
 
   const isAdmin = useAdmin()
   const user = useUser()
+  const answers = useAnswersCpmm(contract.id) ?? contract.answers
+  const isAdminorMod = useAdminOrMod()
+
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [error, setError] = useState<string | undefined>(undefined)
+
+  const onCancelAll = async () => {
+    setIsSubmitting(true)
+    try {
+      for (const answer of answers) {
```

Comment:
Ask chat gpt to turn this for loop into a Promise.all. This will make it only a bit faster because the server will still resolve each answer sequentially.

Reply:
Won't that hit by-IP rate limit?

Reply:
Good question! I don't know our exact limits, but I think a hundred requests in a short period would be ok.

Reply:
I did this (see recent commit), but I think it has introduced an API error.

APIError: High volume of requests. Please try again later.

It managed to get through about 3-4 N/As each time before it then failed to resolve the rest.